lib/heroku/command/pg_backups.rb

```diff
@@ -397,7 +397,9 @@ def unschedule_backups
     attachment = generate_resolver.resolve(db, "DATABASE_URL")
 
     schedule = hpg_client(attachment).schedules.find do |s|
-      attachment.name =~ /#{s[:name]}/
+      # attachment.name is HEROKU_POSTGRESQL_COLOR
+      # s[:name] is HEROKU_POSTGRESQL_COLOR_URL
+      /#{attachment.name}/ =~ s[:name]
```

Comment:
Can you reverse this check to `s[:name] =~ /#{attachment.name}/`? I think that's semantically the same, but more idiomatic (I've never seen a regex on the left of `=~` although it does seem to work).